lighthouse-core/gather/driver.js

```diff
@@ -123,6 +113,11 @@ class Driver {
 
     /** @type {Fetcher} */
     this.fetcher = new Fetcher(this);
+
+    this._runtimeController = new RuntimeController(this);
+
+    /** @type {LH.Gatherer.FRTransitionalDriver} */
+    const typecheck = this; // eslint-disable-line no-unused-vars
```

Comment:
This can't be done on the `class` statement?

Reply:
Yes, `@implements` is supported now: https://github.com/GoogleChrome/lighthouse/blob/165e4924fd60e75ce58a3e5f3ac91ffc3f777f44/lighthouse-core/config/config.js#L300-L303

Reply:
@brendankenny said there were things on the way to have new solutions but I'm not sure if we're there yet. Are we Brendan? :)